spec/static/spec/v0.9/openapitransactional.yaml

```diff
@@ -1950,27 +2004,318 @@ components:
           - http://www.opengis.net/def/crs/EPSG/0/4326
         example:
           title: The new title of the collection
-    roles:
-      type: array
-      items:
-        type: string
-      description: Purposes of the asset
-      example:
-      - thumbnail
-    title:
-      type: string
-      description: Displayed title
-      example: Thumbnail
-    type:
-      type: string
-      description: Media type of the asset
-      example: image/tiff; application=geotiff
     itemIdUpdate:
       description: >-
         Item identifier (unique per collection. If it doesn't match the `featureId`
         in path parameters, then the Item is renamed.
       example: smr200-200-4-2019
       type: string
+    uploadId:
+      title: ID
+      type: string
+      description: Unique Asset upload identifier
+      example: KrFTuglD.N8ireqry_w3.oQqNwrYI7SfSXpVRiusKah0YigDnuM06hfJNIUZg4R_No0MMW9FLU2UG5anTW0boTUYVxKfBZWCFXqnQTpjnQEo1K7la39MYpjSTvIbZgnG
+      readOnly: true
+    dtUploadCreated:
+      title: created
+      description: Date time when the Asset's upload has been created/started.
```

Comment:
we often use `datetime` as one word, and here and elsewhere also as two separate words. Might make sense to be consistent here

Reply:
Well here it is a description where we refer to date and time, I'll add `and`. When we refer to `datetime`, it should be a reference to the `datetime` type. Same same but different :-)